src/javascripts/components/cookies-page.js

```diff
@@ -0,0 +1,174 @@
+import { getConsentCookie, setConsentCookie } from '../cookie-functions.js'
+import common from 'govuk-frontend/govuk/common'
+
+const nodeListForEach = common.nodeListForEach
+
+function CookiesPage ($module) {
+  this.$module = $module
+}
+
+CookiesPage.prototype.init = function () {
+  this.$cookiePage = this.$module
+
+  if (!this.$cookiePage) {
+    return
+  }
+
+  this.$cookieForms = this.$cookiePage.querySelectorAll('.js-cookies-page-form-wrapper')
```

Comment:
I had assumed there'd only being one form around all of the radios on the page, but I don't think I have an issue with this approach if it makes things easier.

Reply:
Actually – one advantage of wrapping the whole lot in a single form is that we can make the 'Save cookie settings' button a submit button for the form, and hook into the normal form submission events rather than the button onClick event.

This'd make the form work consistently with other forms – at the minute, for example, you can't submit the form by pressing enter when focused on a radio button, which you normally can do in a form.